app/views/ViewDates.scala

```diff
@@ -18,10 +18,22 @@ package views
 
 import java.time.ZoneId
 import java.time.format.DateTimeFormatter
+import java.time.temporal.TemporalAccessor
 
-object ViewDates {
+import javax.inject.{Inject, Singleton}
+
+@Singleton
+class ViewDates @Inject()() {
```

Comment:
What are you injecting?

Reply:
Nothing here, this give you possibility to inject ViewDates in other places

Reply:
So it makes the class injectable without having to register it with a module?  Nice.
Should you not remove the DateTimeFormatterProvider from DateTimeModule?  In case it is being used anywhere else?

Reply:
yeah, I'll remove this

Reply:
@PatrykRudnicki You don't need `@Inject` annotation to be able to inject the class into another one (but it has to be `class`, not `object`). You can inject any class you like, for e.g. `Decoder` is injected in many Converter classes, but it has no `@Inject` annotation.
`DateTimeFormatterProvider` was enough some time ago, but since we have a dedicated class for this purpose, it makes sense to remove it.

Reply:
Ah, I thought that Inject is necessary.

Thanks, I'll change it on Monday 🙂

Reply:
I've changed it on my movements branch and it still works!